### ISSUE TITLE
Get all the beacons of all the region in IOS

### ISSUE THREAD

**Issue by AdnanKazi** [alann-maulana/flutter_beacon#32]
I can't detect all the regions that i want 
for example i have 2 regions and i want to find it then i start ranging 

`START RANGING=[{identifier=ibeacon, proximityUUID=99DF7F93-CB6E-4BB0-BE7F-335C373DC416}, {identifier=ibeacon, proximityUUID=B9BA7316-915A-449F-A8D3-512242D6A92D}]` 

but the ranging result shows beacons available on last region.

`RangingResult{
"region": {"identifier":"ibeacon","proximityUUID":"b9ba7316-915a-449f-a8d3-512242d6a92d"}, "beacons": [{"proximityUUID":"B9BA7316-915A-449F-A8D3-512242D6A92D",
"major":3,"minor":38,"rssi":-56,"accuracy":0.59,"proximity":"Proximity.near","txPower":-59,"macAddress":"AC:23:3F:2C:82:8E"}]}
I/flutter (32741): `

Please help me out i just want to scan multiple regions at a time so i can find beacons of different region.

```
I just want all the beacons of all proximityUUID available in area for IOS. Like in android just by
 entering
 `regions.add(Region(identifier: 'ibeacon'));`
```

**Comment by mdanics** [comment]
Did you find a solution to this @AdnanKazi ?

**Comment by AdnanKazi** [comment]
@mdanics Does this problem still occurring ? 
I had found this plugin was not working in ios properly i had change platform I have created that project in React Native 
Sorry I can't help may be in future someone can fix it till that we need to manage.

**Comment by alann-maulana** [comment]
TLDR; No, you cannot scan all UUID on iOS like Android do.

Check an answer from David G. Young from StackOverflow here https://stackoverflow.com/a/37165388/3733730

**Comment by AdnanKazi** [comment]
> TLDR; No, you cannot scan all UUID on iOS like Android do.
> 
> Check an answer from David G. Young from StackOverflow here https://stackoverflow.com/a/37165388/3733730

Thats right
